editor/src/core/shared/element-path-tree.ts

```diff
@@ -159,67 +167,93 @@ function getReorderedPaths(
   metadata: ElementInstanceMetadataMap,
   missingParents: ElementPath[],
 ): ElementPath[] {
-  const elementsToReorder = original.filter((path) => {
-    const element = MetadataUtils.findElementByElementPath(metadata, path)
-    if (element == null) {
-      return false
-    }
-    return (
-      MetadataUtils.isConditionalFromMetadata(element) ||
-      MetadataUtils.isExpressionOtherJavascriptFromMetadata(element) ||
-      MetadataUtils.isJSXMapExpressionFromMetadata(element) ||
-      MetadataUtils.isIdentifierFromMetadata(element) ||
-      MetadataUtils.isPropertyAccessFromMetadata(element) ||
-      MetadataUtils.isElementAccessFromMetadata(element)
-    )
+  const pathsToBeReordered = original.filter((path) => {
+    // Omit elements that have a missing parent.
```

Comment:
The code and the comment is not consistent with each other: comment says missing parent, but code checks for missing ancestor

Reply:
Fixed in 8c91af726.